pkg/models/operational_solution.go

```diff
@@ -37,6 +37,54 @@ func NewOperationalSolution(createdBy uuid.UUID, operationalNeedID uuid.UUID) *O
 	}
 }
 
+func NewOperationalSolutionFull(
```

Comment:
Similar to NewModelPlanFull -- I think we can get rid of this method now that it's not used in the storage methods anymore!